client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -48,50 +59,86 @@ const Settings = ( props: SettingsPageProps ): JSX.Element => {
 		return <Breadcrumbs items={ items } mobileItem={ mobileItem } />;
 	};
 
+	const renderDetailsSection = () => {
+		if ( domain.type === domainTypes.REGISTERED ) {
+			return (
+				<Accordion
+					title={ translate( 'Details' ) }
+					subtitle={ translate( 'Registration and auto-renew' ) }
```

Comment:
You can narrow the returned type and make it a `string` by adding this optional param:

```suggestion
					title={ translate( 'Details', { textOnly: true }  ) }
					subtitle={ translate( 'Registration and auto-renew', { textOnly: true } ) }
```

Thus not needing to change the `Accordion` props' types.

Reply:
Oh, that's a nice one! Thanks for pointing that out 👍